packages/argon2-cffi/test_argon2-cffi.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide(packages=["argon2-cffi"])
+def argon2_cffi_helper(selenium):
```

Comment:
If you're only going to call this function once, you may as well just inline it:
```suggestion
@run_in_pyodide(packages=["argon2-cffi"])
def test_argon2(selenium):
```